auth-web/src/components/auth/manage-business/AddBusinessForm.vue

```diff
@@ -81,7 +81,7 @@
         <v-btn
           large outlined color="primary"
           id="cancel-button"
-          @click="$emit('on-cancel')"
+          @click="resetForm(); $emit('on-cancel')"
```

Comment:
Maybe - emit be moved into reset form?

Reply:
restForm is also being used for adding forms as well as canceling them so I thought it might be best to keep them separate. Although I could add code to differentiate between the two as well?

Reply:
Up to you, just a suggestion (not required) from me